backend-api/src/functions/asyncIssueBiometricCredential/asyncIssueBiometricCredentialHandler.ts

```diff
@@ -513,6 +522,43 @@ export const buildCredentialJwtPayload = (jwtData: {
   };
 };
 
+const writeVCIssuedEvent = async (
```

Comment:
Tiny nitpick RE casing
```suggestion
const writeVcIssuedEvent = async (
```